core/arch/arm/mm/mobj_ffa.c

```diff
@@ -39,7 +39,7 @@ static struct mobj_ffa_head shm_inactive_head =
 
 static unsigned int shm_lock = SPINLOCK_UNLOCK;
 
-static const struct mobj_ops mobj_ffa_ops __rodata_unpaged("mobj_ffa_ops");
+const struct mobj_ops mobj_ffa_ops;
```

Comment:
I don't understand this change, it is not what the commit description says

Reply:
static variables cannot use `__weak` so we must make this non-static, that's mentioned in the commit. Perhaps I should have made this line `extern struct mobj_ops mobj_ffa_ops` instead, but then checkpatch would have complained.
Yes, I'm dropping `__rodata_unpaged("mobj_ffa_ops")` but it's still kept below so that part hasn't really changed. I didn't mention this in the commit since I didn't think it was significant.

Reply:
Oh! It is a forward declaration. How did I miss that? :confused:
Sorry that's fine then.